OmniSharpSublime.sublime-settings

```diff
@@ -3,4 +3,7 @@
     "omnisharp_auto_start": true,
     "omnisharp_server_config_location":"",
     "omnisharp_onsave_codecheck": true,
-}
\ No newline at end of file
+    "omnisharp_onsave_showwarningwindows":true,
+    "omnisharp_onsave_showerrorwindows":true,
+    "omnisharp_debug_log":false

```

Comment:
can you remove this, the output is a bit odd plus its useful for people to see the detail when saying there is a problem :smile: